services/gateway/apis/http-version3/methods/commandsParamsSchemas.js

```diff
@@ -23,14 +23,8 @@ module.exports = {
 	rpcMethod: 'get.commands.parameters.schemas',
 	tags: ['Transactions'],
 	params: {
-		moduleCommandID: { optional: true, type: 'string', min: 3, pattern: /[0-9]+:[0-9]+/ },
-		moduleCommandName: { optional: true, type: 'string', min: 3, pattern: /[a-z]+:[a-z]+/ },
+		moduleCommand: { optional: true, type: 'string', min: 3, pattern: /[a-z]+:[a-z]+/ },
```

Comment:
Please confirm with the SDK team about the allowed characters for the module/command names and adjust the pattern accordingly. And, please move the regex pattern to the `regex.js` file.